### PR TITLE
Minor fix branch

### PR DESCRIPTION
**Pull request by JungAn2**
#28 

- Added comments
- deleted non used library imports
- fixed the exercise service to get exerciseTitle instead of exerciseId
- delete two module files (cyber-security-backend1)
- updated .gitignore to ignore prisma/migration and package-lock.json files

### CODE REVIEW

File: app/services/ExerciseService.ts [hunk 1]
```diff
@@ -69,7 +71,7 @@ export const ExerciseService = {
   /**
    * Gets DataType and splits it up
    *
-   * @param {string} exerciseID id for an exercise
+   * @param {string} s string to conver to datatype
```
Comment (Desss64) [inline app/services/ExerciseService.ts:74]: looks like you forgot the "t" in convert

File: app/services/ExerciseService.ts [hunk 2]
```diff
@@ -69,7 +71,7 @@ export const ExerciseService = {
...
    * @return {any} the data parsed being returned
    */
   getDataType(s: string): any {
```
Comment (Desss64) [inline app/services/ExerciseService.ts:77]: this string parameter could maybe use a more descriptive name

File: app/services/ExerciseService.ts [hunk 3]
```diff
@@ -101,14 +103,17 @@ export const ExerciseService = {
   /**
    * gets status of the exercise
    *
-   * @param {string} exerciseID id for an exercise
+   * @param {string} exerciseTitle id for an exercise
```
Comment (Desss64) [inline app/services/ExerciseService.ts:106]: forgot to replace "id" with "title" in this comment

File: app/services/ExerciseService.ts [hunk 4]
```diff
@@ -122,19 +127,19 @@ export const ExerciseService = {
   /**
    * creates a Database.
    *
-   * @param {string} exerciseID id for an exercise
+   * @param {string} exerciseTitle id for an exercise
```
Comment (Desss64) [inline app/services/ExerciseService.ts:130]: same as previous comment

File: app/services/ExerciseService.ts [hunk 5]
```diff
@@ -147,15 +152,15 @@ export const ExerciseService = {
   /**
    * Fetches content, metadata, and status.
    *
-   * @param {string} exerciseID id for an exercise
+   * @param {string} exerciseTitle id for an exercise
```
Comment (Desss64) [inline app/services/ExerciseService.ts:155]: same as previous

**Review by Desss64 — APPROVED** [review]
There's some small typos here and there but everything else looks good to me

**Review by wrathofrathma — APPROVED** [review]

**Pull request merged**